AElf.Contracts.Authorization/IMessageExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using Google.Protobuf;
+
+namespace AElf.Contracts.Authorization
+{
+    public static class IMessageExtensions
```

Comment:
remove this extension in the future, you should manage null & empty.

it's better not store empty value.